update.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "💾 Saving local changes if any."
+git stash
+
+echo "🪢 Retrieving the latest lively.next version."
+# Assumes that one runs this on `main` branch. Inside of lively, we ensure that the GUI trigger is not shown in other cases.
+git pull
+
+echo "📦 Installing latest version of lively.next."
+./install.sh
+
+# If one has other stashes, they are popped regardless!
+echo "💾 Restoring local changes if any."
+git stash pop
```

Comment:
So if I have some super outdated stash at 0 that conflicts with the current version of lively.next, we basically break the repo here? Can we add a check that only pops if the above stash really stashed files?

Reply:
Done